week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,18 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+
+//var orig = "   foo  ";
+//console.log(orig.trim()); // 'foo'
+
+function tidyUpString(strArr) {
+  var modifiedStr = " ";
+  for (i = 0; i < strArr.length; i++) {
+    strArr[i] = strArr[i].trim().replace("/", "").toLocaleLowerCase();
```

Comment:
This is a neat way of stringing several methods together. Well done! However the method
toLocaleLowerCase() should be
toLowerCase